Makefile

```diff
@@ -7,7 +7,7 @@ DIRS := $(shell find $(SRC) -type d)
 SRCS := $(shell find $(SRC) -type f -name "*.c")
 OBJS := $(SRCS:%.c=$(OBJDIR)/%.o)
 
-CFLAGS = -Iinclude -Wall -Wpedantic -Wextra -Wshadow -std=c11
+CFLAGS = -Iinclude -Wall -Wpedantic -Wextra -Wshadow -std=gnu11
 CFLAGS += -DNDEBUG
```

Comment:
Any reason for this?

Reply:
the removal of line 11: CLAGS += -DNDEBUG

Reply:
I don't remove line 11?